app/views/account/device_groups/_form.html.haml

```diff
@@ -0,0 +1,9 @@
+.form-group
+  .row
+    .col
+      = simple_form_for [:account, @device_group] do |f|
+        .text-left
+          = f.input :name, input_html: { maxlength: 50 }
+        .text-center
```

Comment:
something weird w/ grid construction..
row, col and inside a col 2 divs with different text positioning?